consensus/helpers.go

```diff
@@ -187,11 +186,11 @@ func (m *consensusModule) broadcastToValidators(msg *typesCons.HotstuffMessage)
 		return
 	}
 
+	// Not using Broadcast because this is a direct message to all validators only
```

Comment:
Wdyt about opening an issue to add a `BroadcastStaked()` or `StakedActorBroadcast()` method (or something like that) to the `P2PModule` interface?

It seems to me like `#getValidatorAtHeight()` is doing some of the work of the P2P module (i.e. routing).

Reply:
I've created #849 per your suggestion.

However, remember that this (in consensus) is neither of those as explained in the first ticket you had: #438.